tutorials-dp0-3/portal-dp0-3-3.rst

```diff
@@ -0,0 +1,421 @@
+the oft-observed TNO with ``ssObjectId`` as above.  
+
+.. code-block:: SQL 
+
+    SELECT ra, dec, mag, band, midPointMjdTai 
+    FROM dp03_catalogs_10yr.DiaSource 
+    WHERE ssObjectId = -735085100561880491
+
+
+3.4. The default results view will show the "Coverage" map at upper left.
+In the future, with real LSST data, this map would have an underlay of the LSST deeply stacked image. 
+Since DP0.3 has no images, the "Coverage" map only shows the overlay of RA vs. Dec, which is redundant with the default plot.
+At upper right, click on "Bi-view Tables" to hide the "Coverage" map and show only the table and plot.
+
+.. figure:: /_static/portal_tut03_step03b.png
+    :width: 600
+    :name: portal_tut03_step03b
+    :alt: The default results view after clicking on bi-view tables.
+
+    The "Bi-view Tables" results view for the query of ``ssObjectId`` = -735085100561880491.
+
+
+3.5. Set the color of individual points to represent the time of the observation to 
+better illustrate how the object moves across the sky.
+In the plot panel, click on the "Settings" icon (double gears) to open the "Plot Parameters"
+pop-up window.
+Under "Trace Options", for "Color Map" enter "midPointMjdTai" and for "Color Scale" enter "Rainbow".
+Then click "Apply".
+
+.. figure:: /_static/portal_tut03_step03c.png
+    :width: 600
+    :name: portal_tut03_step03c
+    :alt: A screenshot of the plot of sky coordinates colored as a function of time.
+
+    The 10 loops in the object's path on the sky is a result of Earth's orbital period and the 10-year LSST duration.  
+    Purple color corresponds to earlier observtations, and the red color corresponds to the later observations.  
+
+
+3.6. Clear the query and results and return to the RSP TAP Search form.
+
+.. _DP0-3-Portal-3-Step-4:
+
+Step 4. Plot the time-domain quantities for the TNO
+===================================================
+
+**Note** that no time domain evolution in object brightness was included in the DP0.3 simulation
+(e.g., rotation curves for non-spherical objects, outgassing events).
+All changes in the brightness of DP0.3 objects with time are due to changes in the distance and phase angle from Earth.  
+
+4.1. Execute the following ADQL query to retrieve the r-band magnitudes, phase angles,
+heliocentric and topocentric distances, and time of the observations for the TNO explored in Step 3.
+
+.. code-block:: SQL 
+
+    SELECT ds.midPointMjdTai, ds.mag, ds.band, 
+    ss.phaseAngle, ss.topocentricDist, ss.heliocentricDist 
+    FROM dp03_catalogs_10yr.DiaSource AS ds 
+    JOIN dp03_catalogs_10yr.SSSource AS ss ON ds.diaSourceId = ss.diaSourceId
+    WHERE ss.ssObjectId = -735085100561880491
+    AND ds.band = 'r'
+
+4.2. The default plot will have the r-band magnitude as a function of time.  
+Use the plot "Settings" function to add new scatter plots showing the magnitude and the phase angle as a function of time, ``midPointMjdTai - 60000``.  
```

Comment:
The default plot is already magnitude vs. time, so the user does not need to be adding it like the instructions say. 

Also, the use of midPointMjdTai-60000 is not explained, which makes it seem arbitrary, so add explanation that the reason is so that the number of days along the x-axis becomes clearer.

Reply:
OK